docker-compose/glances.conf

```diff
@@ -595,6 +595,7 @@ queue=glances_queue
 # Configuration for the --export mqtt option
 host=localhost
 port=8883
+hostname=NONE
```

Comment:
Not necessary

Reply:
I Yes I know, but I told myself that it was necessary to put it to show as an example to other users that it was available